Makefile

```diff
@@ -62,6 +62,14 @@ logs: ## Show the containers' logs
 watch: ## Follow the containers' logs
 	${DOCKER_COMPOSE} logs --follow
 
+lint: export TARGET = dev
+lint: ## Lint the code
+	${DOCKER_COMPOSE} run --rm app npm run lint
+
+fix: export TARGET = dev
+fix: ## Fix linting issues in the code
+	${DOCKER_COMPOSE} run --rm app npm run lint:fix
```

Comment:
Have to test this, as it's usually a permission headache on Linux due to the container (running as `node`/`nobody`) possibly lacking the permission to modify the files owned by a different user on the host.

Reply:
Works for me as the `dev` image is running as `root`.